verilog/tools/ls/autoexpand.cc

```diff
@@ -220,6 +222,16 @@ class AutoExpander {
     // Store the given port in the internal vector
     void PutDeclaredPort(const SyntaxTreeNode &port_node);
 
+    // Recurses into dependencies to check if we depend on a given module.
+    // Stores visited modules in a set to avoid infinite loops. For big
+    // dependency graphs one should build a proper graph and do a
+    // topological sort. However, these dependencies (and the dependent) are all
+    // from a single file, and usually there is only one module per file. This
+    // should be fast enough for unusual cases where there are multiple modules
+    // in a single file.
+    bool DependsOn(const Module *module,
+                   absl::flat_hash_set<const Module *> &visited) const;
```

Comment:
Mutable objects like the `visited` shall be passed by pointer not mutable reference.

Reply:
Right, sorry.